app/styles/_sidebar.scss

```diff
@@ -175,7 +175,7 @@ $sidebar-width: 7.8rem;
     -ms-transform: none;
     -o-transform: none;
     transform: none;
-    margin-left: $sidebar-width;
+    margin-left: 5.8rem;
```

Comment:
What was the value of sidebar-width? 
Will this affect on size of sidebar or how it's shown in UI?

Reply:
sidebar-width is 7.8em, 
I have changed it only here, not on the global variable so it will not affect the size of sidebar.